src/utils/utils.js

```diff
@@ -1215,7 +1215,7 @@ export function getDataParentComponent(componentInstance) {
     return;
   }
   const { parent } = componentInstance;
-  if (parent && parent.isInputComponent) {
```

Comment:
I do not see "input" as a property defined on the Component instance. Did you mean to say ```parent.component.input```?

Reply:
This is only for 6.x server, it passes component's schema with 'parent' field which is a parent's schema. So, I just make all the calculations correct calculating path of the component and adding it to that schema, this part now is in the PR for formio, but I'm able to move it here. So, I will update that PR tomorrow.